.github/workflows/on-push-tests.yml

```diff
@@ -0,0 +1,78 @@
+name: Run tests
+on: push
+
+jobs:
+  prepare_dependencies:
+    name: Prepare dependencies
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12"
+
+      - name: Get cached dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            node_modules 
+            packages/vue/node_modules
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install dependencies
+        run: yarn --frozen-lockfile && yarn add @vue/cli -W
+
+  lint:
+    name: Lint
+    needs: prepare_dependencies
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12"
+
+      - name: Get cached dependencies
+        uses: actions/cache@v2
+        with:
+          path: node_modules
```

Comment:
I'm not sure this will work. I'd use `'**/node_modules'` in all 3 cases

Reply:
Ok, I will test this.